x-pack/plugins/alerting/common/alert_schema/scripts/create_schema_from_mapping.ts

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import fs from 'fs';
```

Comment:
Used the event log schema generation script as inspiration. This script ensures that if we make changes to the field mappings for framework alerts, they are reflected in the `Alert` type.